examples/terraform/azure/variables.tf

```
@@ -69,3 +69,12 @@ variable "worker_vm_size" {
   default     = "Standard_B2s"
 }
 
+variable "control_plane_vm_count" {
+  description = "VM Size for worker machines"
+  default     = 3
+}
+
+variable "worker_node_vm_count" {
```

This field is called `initial_machinedeployment_replicas` for AWS. I think that we should keep it consistent for all providers/examples.

```suggestion
variable "initial_machinedeployment_replicas" {
```